marshaling.go

```diff
@@ -348,12 +354,40 @@ func (value *Number) structScanConvert(ctx convertCtx) (reflect.Value, error) {
 	return ctx.cannotConvert()
 }
 
-func (value *Worksheet) structScanConvert(ctx convertCtx) (reflect.Value, error) {
-	return ctx.cannotConvert("not supported yet")
+func (value *Worksheet) structScanConvert(dests map[string]interface{}, ctx convertCtx) (reflect.Value, error) {
+	if ctx.destType.Kind() != reflect.Struct {
+		return ctx.cannotConvert("dest must be a struct")
+	}
+
+	newVal := reflect.New(ctx.destType)
+	dests[value.Id()] = newVal.Interface()
+	err := structScan(dests, value)
+	if err != nil {
+		return reflect.Value{}, err
+	}
+	return newVal.Elem(), nil
 }
 
-func (value *Slice) structScanConvert(ctx convertCtx) (reflect.Value, error) {
-	return ctx.cannotConvert("not supported yet")
+func (value *Slice) structScanConvert(dests map[string]interface{}, ctx convertCtx) (reflect.Value, error) {
+	if ctx.destType.Kind() != reflect.Slice {
+		return ctx.cannotConvert("dest must be a slice")
+	}
+	if len(value.Elements()) == 0 {
+		return reflect.Zero(ctx.destType), nil
+	}
+	locus := reflect.New(ctx.destType)
+	// create backing slice
```

Comment:
not sure this comment is super helpful, and it breaks the method a bit, doesn't feel super natural